app/routes/__index/index.tsx

```diff
@@ -23,11 +21,13 @@ export default function IndexRoute() {
       >
         <Flex
           width={{ base: '100%', md: '75%' }}
-          textAlign="center"
+          maxWidth="42em"
```

Comment:
You could probably edit the values for the width instead of setting a maxWidth. The `base` applies to all breakpoints until the `md` breakpoint. The `md` style applies to all breakpoints including `md` and larger

Reply:
So what should I do?  Can you suggest some code for me?

Reply:
oh, I see you did above!

Reply:
OK, I did your fix above, but I'm not sure how to do the same for this.  If you can recommend something, I'd appreciate it.

Reply:
Instead of maxWidth,
```
width={{ base: '100%', md: '3xl' }}
```
I think that would look better. Currently, the text width is too narrow compared to the cards on smaller sizes:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/67077705/230229216-51229d3a-43a4-4876-b1a0-2e1db509a6dc.png">
After this change,
<img width="447" alt="image" src="https://user-images.githubusercontent.com/67077705/230229329-acc17aa7-e42b-405a-83e1-8915c0f6d18e.png">